src/css/_BRsearch.scss

```diff
@@ -199,6 +169,181 @@
+      width: 14px;
+      height: 14px;
+      margin-left: 7px;
+      content: "";
+      color: var(--primaryTextColor);
+      border: 1px solid var(--primaryTextColor);
+      border-radius: 2px;
+      background: var(--activeButtonBg) 50% 50% no-repeat;
+    }
+      :checked + label.checkbox:after {
+      background-image: url('data:image/svg+xml;base64,PHN2ZyBoZWlnaHQ9IjEwIiB2aWV3Qm94PSIwIDAgMTMgMTAiIHdpZHRoPSIxMyIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIj48cGF0aCBkPSJtNC4zMzMzMzMzMyAxMC00LjMzMzMzMzMzLTQuMTY2NjY2NjcgMS43MzMzMzMzMy0xLjY2NjY2NjY2IDIuNiAyLjUgNi45MzMzMzMzNy02LjY2NjY2NjY3IDEuNzMzMzMzMyAxLjY2NjY2NjY3eiIgZmlsbD0iI2ZmZiIgZmlsbC1ydWxlPSJldmVub2RkIi8+PC9zdmc+');
+    }
+
+    [type="search"] {
+      -webkit-appearance: textfield;
+      width: 100%;
+      height: 22px;
+      padding: 0 15px;
+      box-sizing: border-box;
+      font: normal 16px "Helvetica Neue", Helvetica, Arial, sans-serif;
+      color: var(--primaryTextColor);
+      border: 1px solid var(--primaryTextColor);
+      border-radius: 15px;
+      background: transparent;
+    }
+    [type="search"]:focus {
+      outline: none;
+    }
+    [type="search"]::-webkit-search-cancel-button {
+      width: 18px;
+      height: 18px;
+      -webkit-appearance: none;
+      appearance: none;
+      /* Uses an X SVG shape as a mask */
+      -webkit-mask: url('data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgNDAgNDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgYXJpYS1sYWJlbGxlZGJ5PSJjbG9zZVRpdGxlSUQgY2xvc2VEZXNjSUQiPjxwYXRoIGQ9Ik0yOS4xOTIgMTAuODA4YTEuNSAxLjUgMCAwMTAgMi4xMkwyMi4xMjIgMjBsNy4wNyA3LjA3MmExLjUgMS41IDAgMDEtMi4xMiAyLjEyMWwtNy4wNzMtNy4wNy03LjA3IDcuMDdhMS41IDEuNSAwIDAxLTIuMTIxLTIuMTJsNy4wNy03LjA3My03LjA3LTcuMDdhMS41IDEuNSAwIDAxMi4xMi0yLjEyMUwyMCAxNy44NzhsNy4wNzItNy4wN2ExLjUgMS41IDAgMDEyLjEyMSAweiIgY2xhc3M9ImZpbGwtY29sb3IiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPg==') 0 0 no-repeat;
+      mask: url('data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgNDAgNDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgYXJpYS1sYWJlbGxlZGJ5PSJjbG9zZVRpdGxlSUQgY2xvc2VEZXNjSUQiPjxwYXRoIGQ9Ik0yOS4xOTIgMTAuODA4YTEuNSAxLjUgMCAwMTAgMi4xMkwyMi4xMjIgMjBsNy4wNyA3LjA3MmExLjUgMS41IDAgMDEtMi4xMiAyLjEyMWwtNy4wNzMtNy4wNy03LjA3IDcuMDdhMS41IDEuNSAwIDAxLTIuMTIxLTIuMTJsNy4wNy03LjA3My03LjA3LTcuMDdhMS41IDEuNSAwIDAxMi4xMi0yLjEyMUwyMCAxNy44NzhsNy4wNzItNy4wN2ExLjUgMS41IDAgMDEyLjEyMSAweiIgY2xhc3M9ImZpbGwtY29sb3IiIGZpbGwtcnVsZT0iZXZlbm9kZCIvPjwvc3ZnPg==') 0 0 no-repeat;
+      -webkit-mask-size: 100%;
+      mask-size: 100%;
+      background: #fff;
+    }
+
+    ul {
+      padding: 0;
+      margin: 0;
+      list-style: none;
+    }
+
+    li {
+      display: grid;
+      grid-template-columns: 30px 1fr;
+      grid-gap: 0 5px;
+    }
+
+    li img {
+      display: block;
+      width: 100%;
+    }
+
+    li h4 {
+      grid-column: 2 / 3;
+      padding: 0 0 20px 0;
+      margin: 0;
+      font-weight: normal;
+    }
+
+    li p {
+      grid-column: 2 / 3;
+      padding: 0 0 15px 0;
```

Comment:
Here; this code (+neighbouring) is hard to read because I don't know what `li` represents here

Reply:
I don't understand how this is difficult. These styles are scoped to list items that exist within the parent selector block, which is `#BRsearch_tray`. That selector is nested within `#{$brScope}`. That means that, once compiled, the selector chain for styling search tray list items and their children use the selector chain `.BookReader #BRsearch_tray li, .BRmobileMenu #BRsearch_tray li, .BRfloat #BRsearch_tray li`

Reply:
It's not a blocker; having something like `li.search-result p` would make the CSS self-documenting and easier to read. As is, it's unclear and requires the reader to jump into the HTML code to try decipher where this `li` lives. It also makes the css harder to find from the js/html because there's nothing I can `grep` for to find this css; I'd have to _know_ beforehand that this CSS exists. Or I'd have to try to read all the css in this file to see if any applies to it.

Reply:
It's kind of the equivalent of naming a variable `num`. You know the type, but you don't know what it actually is, and you have to spend time trying to figure out what it is by reading a lot of neighbouring code.

Reply:
This generates more HTML and CSS for the UA to download, when a selector with that level of specificity is not needed. Additionally, this is written to style ANY list items (and certain child elements) with these properties, not just ones that you remember to add a class to. If we're going to apply styles to everything using class names only, we need to first rewrite the HTML and SCSS throughout the project (there are many element selectors throughout the SCSS) and then specify in the contributing guide that CSS is to be applied via class selectors.

On top of all of that, I will reiterate that this is basically throwaway code. As soon as the search results WebComponent is released, it's taking the place of this view constructor and CSS. At that time, nearly all of the CSS selectors within the view will be element-level.